main.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"log"
+	"net/http"
+	"os"
+	"path/filepath"
+)
+
+func main() {
+	// Проверяем наличие пароля в переменной окружения
+	password := os.Getenv("TODO_PASSWORD")
+	if password == "" {
+		log.Fatal("TODO_PASSWORD не задан. Установите пароль в переменной окружения.")
+	}
+
+	// Получаем текущую рабочую директорию
+	wd, err := os.Getwd()
+	if err != nil {
+		log.Fatal("Ошибка получения текущей директории:", err)
+	}
+
+	// Всегда создаём базу данных в корне проекта
+	dbFile := filepath.Join(wd, "scheduler.db")
```

Comment:
На самом деле если сделать так, то ничего не поменяется, БД так же создаться в месте запуска программы
```suggestion
	dbFile := "scheduler.db"
```

Reply:
Я долго возился с тем, что у меня база создавалась во временном окружении при запуске через go run ., внес изменения в код и только таким образом начало работать, даже не было желания еще время тратить на выяснения по этой проблеме, чекну еще раз